src/content/docs/en/guides/routing.mdx

```diff
@@ -18,6 +18,10 @@ Astro uses standard HTML [`<a>` elements](https://developer.mozilla.org/en-US/do
 <p>Read more <a href="/about/">about</a> Astro!</p>
 ```
 
+:::caution
+Changing the [base](/en/reference/configuration-reference/#base) configuration option does not change the `href` of `<a>` elements. This has to be done manually using `Astro.url` or `import.meta.env.BASE_URL` on each element.
+:::
```

Comment:
``````suggestion
<p>Read more <a href="/about/">about</a> Astro!</p>

<!-- With `base: "/docs"` configured -->
<p>Learn more in our <a href="/docs/reference/">reference</a> section!</p> 
```
``````
We only use caution very rarely, for things like security issues. In general, we prefer to document "how something works" properly in the first place, not add notes on like we forgot to say something. :sweat_smile: 

When multiple options are possible, we usually extend the code block to show more examples. I think this is helpful!

Reply:
> We only use caution very rarely, for things like security issues. In general, we prefer to document "how something works" properly in the first place, not add notes on like we forgot to say something. 😅

I will keep that in mind for next time :smiley: , nevertheless, the example shows the opposite of what happens, see https://github.com/withastro/astro/issues/11159.

This is why I thought it was important to explicitly indicate that the anchors are not changed with the base config parameter. For a user coming from other frameworks, they might assume that since the assets are changed by the config, then the anchors must be automatically changed as well.